### PR TITLE
Laser integration

### PR DESCRIPTION
**Pull request by gustavoneves12**
integrated laser line and created a ruby script to test it

### CODE REVIEW

**Comment by gustavoneves12** [comment]
@doudou: PING

**Comment by doudou** [comment]
Sorry for the delay ... I just made some comments on the orogen side ...

**Comment by gustavoneves12** [comment]
@doudou (PING)

File: scenes/worlds/caged_fish.world [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" ?>
```
Comment (doudou) [inline scenes/worlds/caged_fish.world]: The worlds now go in separate directories (scenes/caged_fish/caged_fish.world). We could relax a bit and say that one can do scenes/caged_fish.world, but scenes/worlds really makes no sense.

File: config/orogen/flat_fish_camera_simulation::FlatfishCamera.yml [hunk 2]
```diff
@@ -0,0 +1,63 @@
+# set the camera parameters and laser line parameters
+# to set laser line is necessary to define the link name of the link which represents
+# the laser line and the port that laser scan data will be written
+--- name:camera::front::left
+laser_line_front_params:
+  link_name: 'laser_line_front'
+  pose_port_name: 'laserLineFrontPose'
```
Comment (doudou) [inline config/orogen/flat_fish_camera_simulation::FlatfishCamera.yml]: Is that still actual ? I thought you removed all the dynamic port stuff

File: scenes/worlds/caged_fish.world (hunk 1 above)
Comment (gustavoneves12) [inline scenes/worlds/caged_fish.world]: This world files aren't used anymore.

File: config/orogen/flat_fish_camera_simulation::FlatfishCamera.yml (hunk 2 above)
Comment (gustavoneves12) [inline config/orogen/flat_fish_camera_simulation::FlatfishCamera.yml]: I removed all laser line dynamic ports. But I need to set the link name which represents the laser line in the SDF file.
I forgot to remove the pose_port_name configuration. I made a PR in the flat_fish_camera_simulation [remove_pose_port_name_field] (https://github.com/Brazilian-Institute-of-Robotics/simulation-orogen-flat_fish_camera_simulation/pull/2)

**Comment by doudou** [comment]
I can't merge, there are conflicts with master.

**Comment by gustavoneves12** [comment]
How should I proceed? Can I merge manually and push in the master branch?

**Comment by doudou** [comment]
Merge and push on the PR ...

**Comment by gustavoneves12** [comment]
Done!

**Pull request merged**